relnotes/unregistersocket.migration.md

```diff
@@ -0,0 +1,9 @@
+## IConnectionHandler.unregisterSocket made abstract
+
+* `ocean.net.server.connection.IConnectionHandler`
+
+  `IConnectionHandler.unregisterSocket` method, previously implemented with an
+  empty body is now made abstract, forcing all ConnectionHandler's to
+  unregister socket in a meaningful way before closing them from the finalizer.
+  In turn, `TaskConnectionHandler` now unregister's its `transceiver` before
+  closing it.
```

Comment:
Example of how it may be implemented for apps that didn't have one before?

Reply:
:+1: will update.

Reply:
(I can also spot typo in my text)